http-custom-mechanism/README.adoc

```diff
@@ -271,8 +261,20 @@ Current Principal 'quickstartUser'    </p>
 // Undeploy the Quickstart
 include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
 // Restore the {productName} Standalone Server Configuration Manually
+:restoreScriptName: restore-configuration.cli
 include::../shared-doc/restore-standalone-server-configuration-manual.adoc[leveloffset=+2]
 
+// Additional information about this script
+This script reverts the changes made to the `undertow`  and `elytron` subsystem.You should see the following result when you run the script.
```

Comment:
s/subsystem.You/subsystems. You

Reply:
Another typo I noticed outside of your changes that would be good to correct here is in the "Configure the Server to Use the Custom Module" section:

"This script adds the configuration that enables Elytron security fto use" ->
"This script adds the configuration that enables Elytron security to use"

Reply:
Thanks, I have fixed the spellings